examples/incompressible-flow/2d-sudden-expansion-flow/two-dimensional-jet.geo

```diff
@@ -0,0 +1,84 @@
+// SPDX-FileCopyrightText: Copyright (c) 2025 The Lethe Authors
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception OR LGPL-2.1-or-later
+
+// Variables
+// h : jet height
+// d : distance from bottom
+// Lin : inlet length (development flow)
+// Lout : outlet length
+h = 0.01;
+d = h/2;
+Lin = 8*h;
+Lout = 100*h;
+
+// Mesh parameters
+cell_size = 1.0;
+points_v_Lin = 41; // nodes along x-axis in inlet
+points_v_Lout = 501; // nodes along x-axis in outlet
+points_h_constricted = 11; // nodes along y-axis in constricted section
+points_h_outer = 6; // nodes along y-axis in expanded sections
```

Comment:
If I can give you an advice, it is often better that the coarse mesh be relatively coarse when we can. But in this case it's not a big deal, just for the future :).